frontend/css/sidebar.less

```diff
@@ -74,6 +74,10 @@
   }
 }
 
+.sidebar-fresh-releases {
+  max-height: 85vh;
```

Comment:
```suggestion
  max-height: calc(100vh - @brainzplayer-height - 20px);
```

I think this looks much better. What do you think?

Reply:
Hi
The problem with this is that it works fine when the page loads and gets scrolled a bit down to the first date in the timeline, but when I scroll it up to top and then scroll the sidebar to the bottom, the information filter is hidden again 😅(screenshot attached below).

![image](https://github.com/user-attachments/assets/c227aedb-71a8-4756-9b6a-6ebdd26c1dcc)


Whereas in my solution, this problem does not occur, so I feel that my approach might be a bit better as it displays correctly in the given situation as well (screenshot attached below).
![image](https://github.com/user-attachments/assets/73d92147-561b-4fb0-a6f4-63b47e0602a4)